src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-transition-guide.mdx

```diff
@@ -9,184 +9,199 @@ redirects:
 
 ## Upgrade your monitors [#upgrade]
 
-The synthetics job manager is not legacy runtime compatible. CPMs, on the other hand, are not compatible with the newest runtime. If your job managers and CPMs are sharing a key, then your CPMs will handle any monitors on the legacy runtime, while the job manager will handle any monitors on the newest runtime. In other words, CPMs cannot run scripted monitors that use Chrome 100+ or Node.js 16. 
+The job manager is not legacy runtime compatible and CPMs are not compatible with the newest runtime. CPMs, then, can't run scripted monitors that use Chrome 100+ or Node.js 16. If your job managers and CPMs are sharing a key, then your CPMs will handle any monitors on the legacy runtime, while the job manager will handle any monitors on the newest runtime.
+
+<CollapserGroup>
+        <Collapser
+        id="individual-upgrade"
+        title="Upgrade an individual monitor"
+    >
+    1. Go to <DoNotTranslate>**[one.newrelic.com > Synthetic monitoring](https://one.newrelic.com/synthetics-nerdlets)**</DoNotTranslate>, then select the monitor you want to edit.
+    2. From the <DoNotTranslate>**Settings**</DoNotTranslate> tab, click <DoNotTranslate>**General**</DoNotTranslate>. 
+    3. Switch to the newest runtime view via the dropdown menu. For scripted API monitors, choose Node.js 16. For scripted browser monitors, choose Chrome 100+.
+    4. If using separate private locations for containerized private minions and synthetics job managers, update the location selection.
+    5. Click <DoNotTranslate>**Validate**</DoNotTranslate> to check that your monitors function in the new runtime. Make any script modifications if needed. 
+    </Collapser>
+    <Collapser
+        id="bulk-upgrade"
+        title="Upgrade monitors in bulk"
+    >    
+    The [runtime upgrades UI](/docs/synthetics/synthetic-monitoring/using-monitors/runtime-upgrade-ui/) provides proactive upgrade test results and allows you to bulk upgrade monitor runtimes.
+    </Collapser>
+    <Collapser
+        id="programmatic-upgrade"
+        title="Programmatically upgrade with NerdGraph"
+    >
+    You can refer to our [NerdGraph APIs](/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial/#update-monitors) to programmatically update monitors.
+    </Collapser>
+</CollapserGroup>
+
+If you encounter any problems upgrading your synthetics, refer to [our runtime upgrade troubleshooting guide](/docs/synthetics/synthetic-monitoring/troubleshooting/runtime-upgrade-troubleshooting) for common upgrade errors and solutions.
 
-**Bulk monitor runtime upgrades:**
-The [runtime upgrades UI](/docs/synthetics/synthetic-monitoring/using-monitors/runtime-upgrade-ui/) provides proactive upgrade test results and allows you to bulk upgrade monitor runtimes.
+## Smaller container images for your synthetics [#docker]
```

Comment:
I'm wondering if this section + "Simplified deployment" can be moved into an introductory doc, so we can keep this doc strictly about transitioning

Reply:
If you have any ideas for other docs, or if there's a strong reason to keep this content here, LMK

Reply:
We don't necessarily have a separate doc that introduces the Job Manager. This was included as an introduction to Job Manager for those transitioning from Containerized Private Minion. I think it is useful to highlight changes between the two offerings in the transition guide still, since it serves as both an introduction and a migration guide.